src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/DocumentHelper.java

```diff
@@ -267,6 +279,6 @@ public static BulkPrintDocument caseDocumentToBulkPrintDocument(CaseDocument doc
     }
 
     public List<ContestedConsentOrderData> convertToContestedConsentOrderData(Object object) {
-        return (List<ContestedConsentOrderData>)object;
+        return objectMapper.convertValue(object, new TypeReference<>() {});
```

Comment:
looks like you've made this method more generic? if so, can you update the method name

Reply:
it wasn't about making it generic but about making it actually work - just casting an object to a list didn't work when the object was a map - using `convertValue` did the conversion from a map to a list